packages/contracts-bedrock/src/L1/SuperchainConfig.sol

```diff
@@ -55,25 +56,25 @@ contract SuperchainConfig is Initializable, ISemver {
     /// @param data       Encoded update data.
     event ConfigUpdate(UpdateType indexed updateType, bytes data);
 
-    /// @notice Emitted when a new chain is added as part of the dependency set.
+    /// @notice Emitted when a new dependency is added as part of the dependency set.
     /// @param chainId      The chain ID.
     /// @param systemConfig The address of the SystemConfig contract.
     /// @param portal       The address of the OptimismPortal contract.
-    event ChainAdded(uint256 indexed chainId, address indexed systemConfig, address indexed portal);
+    event DependencyAdded(uint256 indexed chainId, address indexed systemConfig, address indexed portal);
 
-    /// @notice Thrown when the input chain's system config already contains dependencies on its set.
-    error ChainAlreadyHasDependencies();
+    /// @notice Thrown when the dependency set is too large to add a new dependency.
+    error DependencySetTooLarge();
+
+    /// @notice Thrown when the input chain ID is the same as the current chain ID.
+    error InvalidChainID();
 
     /// @notice Thrown when the input chain is already added to the dependency set.
-    error ChainAlreadyAdded();
+    error DependencyAlreadyAdded();
 
     /// @notice Semantic version.
     /// @custom:semver 1.1.1-beta.5
     string public constant version = "1.1.1-beta.5";
```

Comment:
Do you have to upgrade this?

Reply:
No need to because it is updated in the main PR that has not been merged to develop yet